content/en/blog/2023/sig-network-spotlight.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: blog
+title: "Spotlight on SIG Network"
+slug: sig-network-spotlight
+date: ## Add merge date here
+slug: sig-network-spotlight-2023
+author: "Sujay Dey"
+---
+
+Networking is one of the core pillars of Kubernetes and SIG Network, or the Kubernetes Special Interest Group for Networking, is responsible for developing and maintaining the networking features of Kubernetes. It takes care of all aspects to ensure Kubernetes provides a reliable and scalable network infrastructure for containerized applications.
+
+In this SIG Network spotlight, [Sujay Dey](https://twitter.com/Sujaystwt) talked with [Shane Utt](https://twitter.com/ShaneUtt), Software Engineer at Kong, chair of SIG Network and maintainer of Gateway API, on different aspects of the SIG, what are the exciting things going on and how anyone can get involved and contribute.
+
+**Sujay**: Hello, first of all, thanks for the opportunity of learning more about SIG
+Network. I would love to hear your story so could you please tell us a bit
+about yourself, your role, and how you got involved in Kubernetes, especially in SIG Network?
+
+**Shane**: Hello! Thank you for reaching out.
+
+My Kubernetes journey started while I was working for a small data center: we
+were early adopters of Kubernetes and focused on using Kubernetes to provide
+SaaS products. That experience led to my next position developing a distribution
+of Kubernetes with a focus on networking. During this period in my career, I was
+active in SIG Network (predominantly as a consumer).
+
+When I joined [Kong][kong] my role in the community changed significantly, as
+Kong actively encourages upstream participation. I greatly increased my
+engagement and contributions to the [Gateway API][gwapi] project during those
+years and eventually became a maintainer.
+
+I care deeply about this community and the future of our technology, so when
+a chair position for the SIG became available I volunteered my time
+immediately. I've enjoyed working on Kubernetes over the better part of a
+decade and I want to continue to do my part to ensure our community and
+technology continues to flourish.
+
+[kong]:https://konghq.com/
+[gwapi]:https://gateway-api.sigs.k8s.io/
+
+**Sujay**: I have to say, that was a really interesting journey! Now let's talk a bit more about sig-network. Since we know that SIG Network covers a lot of grounds, so could you please highlight its scope and current focus areas?
+
+**Shane**: For those who may be uninitiated: SIG Network is responsible for the components,
+interfaces, and APIs which expose networking capabilities to Kubernetes users
+and workloads. The [charter][net-charter] is a pretty good indication of our
+scope, but I can add some additional highlights on some of our current areas of
+focus:
+
+### KubeProxy / KPNG
```

Comment:
Using level 3 subheadings seems the most adequate solution, but when rendered it looks like it "breaks" the interview, almost as if it's a separate section to it. The final horizontal ruler tries to overcome this, but it adds an additional separation. Have you tried just using something like `**KubeProxy/KPNG**` in a line by itself? I think that this might make the ruler unneeded. Just a thought though.

Reply:
Yeah, that makes it more readable, I should have tried this earlier